mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/domain/MtrainingModuleActivityRecordAudit.java

```diff
@@ -0,0 +1,54 @@
+package org.motechproject.nms.mobileacademy.domain;
+
+
+
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+import org.motechproject.mds.domain.MdsEntity;
+
+import javax.jdo.annotations.Column;
+import javax.jdo.annotations.Unique;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+
+/**
+ * Audit table to track changes in MTRAINING_MODULE_ACTIVITYRECORD
+ */
+@Entity(tableName = "MTRAINING_MODULE_ACTIVITYRECORD_Audit")
+public class MtrainingModuleActivityRecordAudit extends MdsEntity {
```

Comment:
Is there a reason why this is a separate entity? Why not just use the track changes annotation to track this, like other places in code?